README.md

```diff
@@ -5,15 +5,7 @@ BaseDAO is a generic smart contract on Tezos that enables a community to collect
 The contract enables the creator to customize their DAO based on a number of attributes, and uses a ‘decision lambda’ to specify arbitrary code that can be governed by a DAO’s proposals.
 BaseDAO also includes Permit (TZIP-17) to enable off-chain voting.
 
-This repository provides:
-* Generic DAO contract, written in cameLIGO, that can be used for various DAOs implementations.
-* Instantiations of this contract:
-  1. Simplest implementation with no custom logic ([TrivialDAO](#trivialdao))
-  2. Two more practical DAO contracts: [RegistryDAO](#registrydao) and [TreasuryDAO](#treasurydao).
-* A [TypeScript API](#typescript-api) to interact with DAO contracts.
-* [Contract tests](#testing) to validate its implementation.
-
-## Contract documentation and configurations
```

Comment:
This section appear to be removed, and does not appear to be included anywhere else. Was that intentional?

Reply:
Yep, that was intentional.

That section was a quick summary with more details below, but now that the README's details have been moved to other doc files this is no longer necessary.

My idea is that now the README itself is the summary.